jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/OdePropertiesTest.java

```diff
@@ -71,25 +67,6 @@ public void testInit() {
                   fail("Unexpected exception: " + e);
             }
       }
-      
-      // @Test
```

Comment:
If we no longer need this test, should we just remove it?

Reply:
I am confused

Reply:
It still tests setters and getters.

Reply:
I'm...not sure what i was getting at this one.